src/libraries/Common/tests/SingleFileTestRunner/SingleFileTestRunner.cs

```diff
@@ -30,6 +30,21 @@ public static int Main(string[] args)
         // The current RemoteExecutor implementation is not compatible with the SingleFileTestRunner.
         Environment.SetEnvironmentVariable("DOTNET_REMOTEEXECUTOR_SUPPORTED", "0");
 
+        // GH dotnet/runtime Issue #85417/PR #91229: We have some tests that are
```

Comment:
This part of the comment and references to issues is unnecessary. We're going to keep this even if these are fixed. Having a mechanism to exclude test from R2R if there's a future issue is useful.

Reply:
I think having the reference to the issue would help any people working on this file in the future to understand the context behind the change.

Reply:
This can be looked up in the git history. All of these mechanisms to detect various test configurations and platforms (such as `DOTNET_REMOTEEXECUTOR_SUPPORTED` right above) were introduced so that we can do things conditional to the configuration. It is the obvious explanation and it is the correct explanation. What additional context does the issue/PR provide here?

Reply:
It is not obvious to me, and I don't think there's any harm in saving people the time it would take to look up the Git history, so I respectfully disagree. That said, if you have a strong opinion that we really shouldn't write this, then I can remove it, no problem there :+1:

Reply:
The details in the second paragraph of the comment is sufficient for anyone working on this. Feel free to keep the first paragraph but there's a lot of value in succinct code. The PR/issue don't add useful information that wouldn't be fully covered by the second paragraph. I would just replace "So, to achieve this" at the beginning of the second paragraph with "To detect ReadyToRun testing" and delete the first paragrah.